Added api endpoint description to Get application tag Catalog (#104)
instana/openapi: spec/descriptions/getApplicationTagsCatalog.md

> Use this API Endpoint if one wants to retrieve a list of Tags.

Reviewer: ```suggestion
Use this API endpoint to retrieve a list of tags.
```

> It also shows tags which is grouped together. This grouping acts as the dimensionality of an event. Eg: `Call` can have 9 attributes.

Reviewer: ```suggestion
This API endpoint also shows tags which are grouped together. This grouping also denotes the dimensionality of an event. Eg: `Call` can have 9 attributes.
```

Also these tags have cardinality ranging from low to high which will help in examining broad or narrow patterns during the analysis.

> Queryable tags is a powerful enabler for slice and dice analysis in `Query Builder` which is present in `Unbounded Analytics`.

Reviewer: ```suggestion
Queryable tags are a powerful enabler for slice and dice analysis in `Query Builder`.  `Query Builder` can be found in `Unbounded Analytics`.
```
